src/vaadin-dialog-resizable-mixin.html

```diff
@@ -89,6 +90,12 @@
         left: -4px;
         cursor: nwse-resize;
       }
+
+      /* IE11 -only CSS */
+      _:-ms-fullscreen,
```

Comment:
Is this a common way of IE11-only CSS? It looks like the most often used is this one:

```
@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
}
```

Not a blocker, just curious.

Reply:
Right, I picked the IE11-specific selector from https://stackoverflow.com/questions/18907131/detecting-ie11-using-css-capability-feature-detection

The media query also applies to IE10, but that's not a problem in our case. I'll change it.

Reply:
Well, looks like another test (on IE11) started breaking if I use the media query. Wonder if it's about some Shady CSS scoping issue related to media queries. Unless you want to debug more, I'd suggest continuing with the current revision.

Reply:
Sure, the current working version LGTM. Marking as resolved.